x-pack/test/functional/services/transform/index.ts

```diff
@@ -23,10 +23,12 @@ import { TransformAlertingProvider } from './alerting';
 
 import { MachineLearningAPIProvider } from '../ml/api';
 import { MachineLearningTestResourcesProvider } from '../ml/test_resources';
+import { MachineLearningSecurityCommonProvider } from '../ml/security_common';
 
 export function TransformProvider(context: FtrProviderContext) {
   const api = TransformAPIProvider(context);
-  const mlApi = MachineLearningAPIProvider(context);
+  const mlSecurityCommon = MachineLearningSecurityCommonProvider(context);
+  const mlApi = MachineLearningAPIProvider(context, mlSecurityCommon);
```

Comment:
I might be missing something, but looking at `MachineLearningAPIProvider` function, I didn't see a second param being passed in. If mlSecurityCommon is not used, we can revert changes in these two lines.

Reply:
https://github.com/elastic/kibana/pull/179321/commits/bc67dd232791da7c14b3217e735015d3fa3ea36a